README.md

```diff
@@ -81,41 +81,61 @@ By default we download all the libraries from GitHub and build them locally, so
 
 ### Configuring on Windows
 
+#### Installing the dependencies / environment
 Install [MS Visual Studio Community / Enterprise](https://visualstudio.microsoft.com/vs/community/), [Strawberry perl](https://strawberryperl.com/), and [Chocolatey](https://chocolatey.org/install) if not installed already
```

Comment:
Am I able to use `winget` instead of `chocolatey`?

https://learn.microsoft.com/en-us/windows/package-manager/winget/

Reply:
We recommend using choco